editor/register_editor_types.cpp

```diff
@@ -265,6 +265,7 @@ void register_editor_types() {
 
 	GLOBAL_DEF(PropertyInfo(Variant::STRING, "editor/script/templates_search_path", PROPERTY_HINT_DIR), "res://script_templates");
 
+	GLOBAL_DEF("editor/naming/function_parameter_prefix", ""); // Like "p_".
```

Comment:
Could this be a String enum? As in, letting you pick from a list but still allowing custom strings. I have never seen a Project Setting do this, but because the `PROPERTY_HINT_ENUM_SUGGESTION exists`, it does not sound impossible.